### ISSUE TITLE
get error in 20.04.2-Ubuntu

### ISSUE THREAD

**Issue by skk294** [ekvedaras/redis-gui#31]
```
:/opt$ ./redis-gui-1.2.8.AppImage 
(node:7282) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
A JavaScript error occurred in the main process
Uncaught Exception:
/tmp/.mount_redis-L3n0JN/resources/app.asar/src/background.js:3
import { app, BrowserWindow, protocol } from 'electron'
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1060:16)
    at Module._compile (internal/modules/cjs/loader.js:1108:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)
    at Module.load (internal/modules/cjs/loader.js:992:32)
    at Module._load (internal/modules/cjs/loader.js:885:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12633)
    at Object.<anonymous> (electron/js2c/browser_init.js:185:3510)
    at Object../lib/browser/init.ts (electron/js2c/browser_init.js:185:3714)
    at __webpack_require__ (electron/js2c/browser_init.js:1:128)
    at electron/js2c/browser_init.js:1:1200

```

**Comment by ekvedaras** [comment]
Probably need to fully define `snapcraft.yml`. Sadly the old way of publishing no longer works :/ Will investigate. If someone has experience with this and can provide a PR or at least some guidance that would be great 👍

**Comment by ekvedaras** [comment]
Should be fixed in `v1.2.9`
